docs/gl_objects/pipelines_and_jobs.rst

```diff
@@ -274,6 +274,18 @@ You can also directly stream the output into a file, and unzip it afterwards::
     subprocess.run(["unzip", "-bo", zipfn])
     os.unlink(zipfn)
 
+It is also possible to use the underlying iterator directly::
+
+    artifact_bytes_iterator = build_or_job.artifacts(streamed=True, action='iterator')
+
+This can be used with FastAPI/Starlette StreamingResponse to forward a download from gitlab without having to download
+the entire file server side first::
```

Comment:
Maybe this would sound slightly more generic?

```suggestion
This can be used with frameworks that expect an iterator (such as FastAPI/Starlette's
``StreamingResponse``) to forward a download from GitLab without having to download
the entire content server-side first::
```

Reply:
Same here!